data_pipeline/glove_embeddings.py

```diff
@@ -6,7 +6,7 @@
 import numpy as np
 import pickle
 
-GLOVE_DATA_FILE = 'glove.6B.50d.txt'
+GLOVE_DATA_FILE = 'glove.6B.300d.txt'
```

Comment:
I'd advice you to revert this as the PR with generalization will cause conflicts for this. It's okay that you have tried it locally as well, but I think it would be easier for you afterwards, as you will only need to change GLOVE_EMB_DIM from config, and that's it.

Reply:
I sense there will be some conflicts, changed it back.